packages/react-components/review/beta/react-components.api.md

```diff
@@ -261,6 +261,7 @@ export interface ComponentStrings {
     cameraButton: CameraButtonStrings;
     devicesButton: DevicesButtonStrings;
     dialpad: DialpadStrings;
+    DomainPermissions: DomainPermissionsStrings;
```

Comment:
It's weird that an `@internal` component has strings in `@beta`.

If it's truly an `@internal` component, then Contoso never needs to override strings.
Ditto for icons.

Reply:
The strings need to be beta for the localization provider but we are keeping the component internal for now until we understand how exactly we want to expose the components individually.

Reply:
Made Beta 😊